packages/extension/src/config.ts

```diff
@@ -1224,6 +1228,51 @@ export const EmbedChainInfos: ChainInfo[] = [
     },
     beta: true,
   },
+  {
+    rpc: EVMOS_RPC_ENDPOINT,
+    rpcConfig: EVMOS_RPC_CONFIG,
+    rest: EVMOS_REST_ENDPOINT,
+    restConfig: EVMOS_REST_CONFIG,
+    chainId: "evmos_9000-2",
+    chainName: "Evmos",
+    stakeCurrency: {
+      coinDenom: "PHOTON",
+      coinMinimalDenom: "aphoton",
+      coinDecimals: 18,
+    },
+    walletUrl:
+      process.env.NODE_ENV === "production"
+        ? "https://wallet.keplr.app/#/evmos/stake"
+        : "http://localhost:8080/#/evmos/stake",
+    walletUrlForStaking:
+      process.env.NODE_ENV === "production"
+        ? "https://wallet.keplr.app/#/evmos/stake"
+        : "http://localhost:8080/#/evmos/stake",
+    bip44: {
+      coinType: 118,
```

Comment:
Note this uses the `eth_secp256k1` as the default key for address derivation
```suggestion
      coinType: 60,
```